docs/creating-manifest/events.md

```diff
@@ -1757,7 +1757,93 @@ onAfterRemoveSSL:
 ```
 @@!
 
+### onBeforeInstallAddon  
+The event is executed before add-on installation  
 
+Event Placeholders:  
+
+-   `${event.params.}`:  
+    - `settings` - custom form with predefined user input elements  
+    - `id` - add-on id  
+    - `lang` - [user's language code](https://en.wikipedia.org/wiki/Language_code) which was used during the installation. Optional parameter  
+    - `region` - region, where an environment will be installed  
+    - `targetAppid`  - application identifier  
+-   `${event.response.}`: parameters are absent  
+
+### onAfterInstallAddon  
+The event is executed after add-on installation  
+  
+-   `${event.params.}`:  
+    - `settings` - custom form with predefined user input elements   
+    - `id` - add-on id  
+    - `lang` - [user's language code](https://en.wikipedia.org/wiki/Language_code) which was used during the installation. Optional parameter  
+    - `region` - region, where an environment will be installed  
+    - `targetAppid`  - application identifier  
+-   `${event.response.}`:  
+    - `uniqueName` - installed add-on unique name  
+    - `appid` - unique environment *appid* at the Jelastic Platform  
+    - `successText`- message that appears once action is successfully performed  
+  
+Subscription:  
+  
+@@@
+```yaml
+type: update  
+name: Test Add-on Events Subscription  
+  
+onBeforeInstallAddon [nodeGroup:cp, id:test123]:  
```

Comment:
replace *test123* with *yourAddonId* (all occurrences)

Reply:
I replaced addon id 'test123' with 'myaddon'. 
And I propose to add a 'unique' word in front of add-on id for better understanding that add-ons' ids cannot coincide